src/components/comment-input.tsx

```diff
@@ -1,9 +1,10 @@
-import { useUser } from "@auth0/nextjs-auth0";
 import React, { ChangeEvent } from "react";
 import styled from "styled-components";
 
 import { useComments } from "../providers/comments-provider";
-import { Comment } from "../types";
+import { useUser } from "../providers/user-profile-provider";
+import { postComment } from "../requests/comments";
```

Comment:
Going forward, we can see if we can restructure our files so that folders focus on features rather than implementation details. A provider for X is closer to other X-related matter rather than to providers Y and Z. Other providers can even live on the different branches of a fractal tree, if the tree is big enough.

This is similar to how ``` Wrapper = styled.div`` ``` is closer to a given component rather than a folder called `css` with _all_ styles for the project.

We should be good for now; this is something to think about as we grow the codebase.

Reply:
Sounds reasonable.